src/js/schemas/job-schema/Schedule.js

```diff
@@ -59,16 +61,17 @@ const Schedule = {
       }
     },
     timezone: {
-      title: "Time Zone",
+      title: i18nMark("Time Zone"),
       description: (
         <span>
-          {"Enter time zone in "}
+          {i18nMark("Enter time zone in ")}
```

Comment:
`i18nMark` is not translating anything here. Since this is already a jsx element, it would make sense to translate using `Trans` here, by removing `i18nMark`s and using `<Trans render="span">` in place of `<span>` on line 66